src/init/starship.nu

```diff
@@ -12,7 +12,19 @@ let-env PROMPT_COMMAND = {
     ^::STARSHIP:: prompt $"--cmd-duration=($env.CMD_DURATION_MS)" $"--status=($env.LAST_EXIT_CODE)" $"--terminal-width=($width)"
 }
 
-# Not well-suited for `starship prompt --right`.
-# Built-in right prompt is equivalent to $fill$right_format in the first prompt line.
-# Thus does not play well with default `add_newline = True`.
-let-env PROMPT_COMMAND_RIGHT = {''}
+# The minimum required version is 0.71.0
+let has_right_prompt_support = ((version).version >= 0.71.0)
+
+# Whether we enable the config
+let config_enabled = (($env | get -i config.render_right_prompt_on_last_line) == true)
```

Comment:
I would prefer to attempt to set `render_right_prompt_on_last_line` to `true`, and to allow users to set it to `false` based on their preferences and keep the rprompt.

Reply:
So we just need to delete this line?

Reply:
Shall we set it to true in `starship.nu`？

Reply:
> and to allow users to set it to `false` based on their preferences and keep the rprompt.

Seems there's no need to check version too :joy:

Reply:
Yes.